### PR TITLE
A0-3911 Bump versions and update CHANGELOG

### PR DESCRIPTION
**Pull request by deuszx**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## 0.7.0
+
+
```
Comment (obrok) [inline CHANGELOG.md:12]:
```suggestion

```

File: ink-wrapper-types/Cargo.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "ink-wrapper-types"
-version = "0.6.0"
+version = "0.7.0"
```
Comment (obrok) [inline ink-wrapper-types/Cargo.toml:3]: Suggestion: bump both to `0.8.0` now?

**Review by obrok — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (deuszx) [reply CHANGELOG.md]: Done.

File: ink-wrapper-types/Cargo.toml (hunk 2 above)
Comment (deuszx) [reply ink-wrapper-types/Cargo.toml]: And done.

**Pull request merged**